pmoves/services/hi-rag-gateway-v2/app.py

```
@@ -1801,7 +1918,10 @@ def geometry_decode_image(body: Dict[str, Any], _=Depends(require_tailscale)):
         if mode == "swarm":
             sidecar = _get_gan_sidecar()
             accept_threshold = float(body.get("accept_threshold", 0.55))
-            max_edits = int(body.get("max_edits", 0))
+            try:
+                max_edits = int(body.get("max_edits", 0))
+            except (TypeError, ValueError):
+                raise HTTPException(400, "max_edits must be an integer")
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)